Description: Bump number of attempts to start SKS mock keyserver
 Sometimes, the SKS mock keyserver didn't started within five attempts /
 five seconds, for example, due to high load of the machine.
 Accordingly, increase the number of attempts to start it and the
 seconds to sleep in between each try.

 This is already fixed upstream, this patch will be dropped when
 uploading the next upstream release.

Author: Georg Faerber <georg@riseup.net>
Last-Update: 2018-02-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -80,9 +80,9 @@
   def with_sks_mock
     pid = Process.spawn('spec/sks-mock.rb', [:out, :err] => ["/tmp/sks-mock.log", 'w'])
     uri = URI.parse("http://127.0.0.1:9999/status")
-    attempts = 5
+    attempts = 25
     begin
-      sleep 1
+      sleep 5
       Net::HTTP.get(uri)
     rescue Errno::ECONNREFUSED => exc
       attempts -= 1
